internal/ethapi/api_test.go

```diff
@@ -361,7 +361,7 @@ func allTransactionTypes(addr common.Address, config *params.ChainConfig) []txDa
 	}
 }
 
-func allBlobTxs(addr common.Address, config *params.ChainConfig) []txData {
+func allBlobTxs(addr common.Address, _ *params.ChainConfig) []txData {
```

Comment:
same here

Reply:
yes, it is a bit unnecessary. And test ci has already checked this functions. I have restored this functions.